### DISCUSSION TITLE
Hard time with Counter metrics

### DISCUSSION THREAD

**Discussion by nicobleiler** [spatie/laravel-prometheus#47] [Q&A]
I used the default setup and only added some basic gauges and wanted to add some counters, like a login count.
Without adding setInitialValue it does not even show up unless I were to increase it at the same time, also I was unable to figure out how to access the counter in other parts to increase it.
I set the config `'cache' => 'redis',`
```
        /*
        * Define the login counter metric
        */
        Prometheus::addCounter('Login count')->helpText('Total number of user logins')->setInitialValue(0);
```

**Comment by DennisHoefer** [comment]
Facing the issue. A clarification would be greatly appreciated 🙏🏻
